server/api/orders.js

```diff
@@ -21,7 +22,32 @@ router.get('/:id/totalCost', (req, res, next) => {
     .then(totalCost => {
       res.json(totalCost)
     })
+})
 
+router.delete('/:id/:itemId', (req, res, next)=> {
+  Order.findById(req.params.id)
+    .then(order => {
+      order.removeProduct(req.params.itemId)
+      res.sendStatus(200);
+    })
+    .catch(console.error);
 })
 
+router.put('/:id/:itemId', (req, res, next)=> {
+  Order_Product.update({
+    quantity:req.body.quantity
+  },{
+    where: {
+      productId: req.params.itemId
+    }
+  })
+    .spread((num, order) => {
+      if (num) return res.status(200).json(order);
```

Comment:
I believe that we still need to send something back either way here, so as to not get an error.